test/unit/comments.spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as util from "../util";
+
+test("JSDoc is copied on a function", () => {
+    const builder = util.testModule`
+        /**
+         * This is a function comment.
+         * It has multiple lines.
+         */
+        function foo() {}
+    `.expectToHaveNoDiagnostics();
```

Comment:
Can we have a snapshot test for at least one of these or a similar snippet? Usually, we try to avoid them, but in this context a snapshot test seems useful to me.

Reply:
im not sure how to do that, can you help?

Reply:
Simply add `.expectToMatchSnapshot()` to the test builder here

Reply:
ok, its committed, let me know if i did it right